source/_integrations/pi_hole.markdown

```diff
@@ -22,10 +22,23 @@ ha_integration_type: integration
 ---
 
 The Pi-hole integration allows you to retrieve statistics and interact with a
-[Pi-hole](https://pi-hole.net/) system. If your Pi-hole web interface is password protected, an API key is needed for the setup (_from Settings -> API / Web interface_).
+[Pi-hole](https://pi-hole.net/) system.
 
 {% include integrations/config_flow.md %}
 
+During the setup, it will ask for the following:
+
+| Item | Description | Example |
+| ---- | ----------- | ------- |
+| `Host` | The IP or domain name to Pi-Hole | 192.168.1.1 |
+| `Port` | Port used to get to the admin page | 80 |
+| `Name` | Name to for this Pi-Hole. | Pi-Hole |
+| `Location` | the path to the admin page. | /admin |
+
+The combined host, port and location should take you to the login page of Pi-Hole. Using the example above, it would be [http://192.168.1.1:80/admin](http://192.168.1.1:80/admin)
```

Comment:
We should not make local links on public websites.

The issue with those is that search engines/crawlers may try to follow those.

```suggestion
The combined host, port, and location should take you to the login page of Pi-Hole. Using the example above, it would be `http://192.168.1.1:80/admin`.
```